packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -602,15 +614,26 @@ export class StatsAnalyzer extends EventsScope {
             `StatsAnalyzer:index#compareLastStatsResult --> No video RTP packets sent`,
             currentStats.totalPacketsSent
           );
-        } else {
+        } else if (this.lastEmittedStartStopEvent.video.local !== EVENTS.LOCAL_MEDIA_STOPPED) {
```

Comment:
previously the code would output the log "No video Frames Encode" every 5s for as long as we were not encoding the frames and now it won't do that (because even if you set lastEmittedStartStopEvent.video.local to EVENTS.NO_VIDEO_ENCODED it will get overwritten to LOCAL_MEDIA_STOPPED by emitStartStopEvents that gets called later)  - this is not good, having those logs is useful

Reply:
Removed the condition from here. This was done to avoid sending multiple metrics to amplitude and added the condition before sending the event. So now logs will be printed after every 5s and the event will only be sent once.